lib/appraisal/bundler_dsl.rb

```diff
@@ -100,22 +100,28 @@ def ruby_version_entry
 
     alias_method :ruby_version_entry_for_dup, :ruby_version_entry
 
-    [:dependencies, :gemspec].each do |method_name|
-      class_eval <<-METHODS, __FILE__, __LINE__
-        private
+    def gemspec_entry
+      if @gemspecs
```

Comment:
Should this be `@gemspecs.any?` or something? Do we still need a conditional now that this will never be `nil`?